src/pybind/mgr/cephadm/module.py

```diff
@@ -986,6 +989,8 @@ def _refresh_host_services(self, host):
         out, err, code = self._run_cephadm(
             host, 'mon', 'ls', [], no_fsid=True)
         data = json.loads(''.join(out))
+        for d in data:
+            d['last_refresh'] = datetime.datetime.utcnow().strftime(DATEFMT)
```

Comment:
Why not [`.isoformat()`](https://stackoverflow.com/a/28147286/3185053)?

Reply:
Oh, didn't know about that.  I wonder if it's better to be explicit, though, because we're both generating a string and parsing it with strptime?